js/components/Verse.react.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Verse
+ */
+
+import { asyncEnableRecall, asyncDisableRecall } from '../actions/AppActions';
+
+import React, { Component } from 'react';
+
+class Verse extends Component {
+  render() {
+    const { index, book, chapter, text, verse, isRecalling, dispatch } = this.props;
+
+    function splitText(text) {
+      return text.split(' ').map(function (word) {
+        return word[0] + word.slice(1, word.length).replace(/\w/g, ' ');
+      }).join(' ');
+    }
+
+    return (
+      <li>
+        <div className="passage-card">
+          <p className="passage-metadata">{ book + ' ' + chapter + ':' + verse }</p>
```

Comment:
Not sure if this based on the latest changes in `master`, but one reason that I had `verseMetadata` as a property in the state was so that a verse could eventually be passed multiple verses.  Was thinking that it would useful for showing verses grouped by week pace or other ways we may want to include verses together